### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by eriktaubeneck**
updating the README and Cargo.toml as I'm getting started. 

A few of these benchmarks aren't working for me, and I added notes in the readme. I can delete those and open issues, if that's preferred, but I just wanted to make sure we documented it somewhere.

### CODE REVIEW

File: README.md
Comment (akoshelev) [inline README.md:88]: there is a fix for it #754

File: README.md
Comment (akoshelev) [inline README.md:94]: Fixed by #754 as well

File: README.md
Comment (akoshelev) [inline README.md:99]: this is expected if you're using MacOS - to the best of my knowledge valgrind does not support it properly. `iai` framework uses it to run the test. We could add a line that this only works on Linux

File: README.md
Comment (akoshelev) [inline README.md:33]: `git clone https://github.com/private-attribution/ipa` works too

**Review by akoshelev — APPROVED** [review]
lets merge #754 and make sure commands work before merging this to main

**Comment by eriktaubeneck** [comment]
Updated this. Should we wait until #754 is merged until we merge this?

**Review by akoshelev — APPROVED** [review]

**Pull request merged**
